opl/hbi_utils.py

```diff
@@ -105,46 +105,43 @@ def wait_for_next_second():
             f"Not all messages sent {data_stats['successes']} + {data_stats['failures']} != {args.count}"
         )
 
+        
+def fetch_records_count(inventory):
```

Comment:
Please do not mix this cluster_read change with hbi_utils change in one PR.

Reply:
This code exists in master branch no changes required in this merge request.

Reply:
So it should not be in the PR. Please rebase your PR.